lms/djangoapps/instructor/views/api.py

```diff
@@ -3407,6 +3407,7 @@ def update_cohort_assignment(request, course_id):
 @require_http_methods(['POST', 'GET'])
 def cohorts_list_with_assignment(request, course_id):
     cohort_info = json.loads(cohort_handler(request, course_id).content)
+    cohorts = cohort_info.get('cohorts', list())
```

Comment:
I would use a literal empty list creation instead of the constructor.

Reply:
No objections really. It always depends on a developer. The literal representation actually even takes fewer lines in byte code. This is a plus to this approach. I'll redo.

Reply:
Done